examples/analytics_specs/main.tf

```diff
@@ -1,5 +1,6 @@
 module "atlas_basic" {
-  source = "../../"
+  source = "terraform-mongodbatlas-modules/atlas-basic/mongodbatlas"
+  version = "0.0.1"
```

Comment:
does the version have to be hardcoded? Thinking if we can keep this more generic to avoid having to update constantly to latest version

Reply:
i think the only option here would be not to declare a version. otherwise i think it is still not possible to declare the version with a variable or something like that https://github.com/hashicorp/terraform-provider-tfe/issues/849.

Maybe we should put it like this adding the beta suffix? 
https://developer.hashicorp.com/terraform/language/expressions/version-constraints#version-constraint-behavior

Reply:
I tried without declaring `version` and it worked. I think it is better option than having to hard-code it every time. WDYT? @AgustinBettati

Reply:
Yes, I also tried it without version and it indeed works. I updated the examples skipping this attribute.